docs/config-app.md

```diff
@@ -32,6 +32,15 @@ This parameter affects how many CPU cores will be utilized by the application. R
 - `http-client.use-compression` - if equals to `true` httpclient compression is enabled for requests (see [also](https://vertx.io/docs/apidocs/io/vertx/core/http/HttpClientOptions.html#setTryUseCompression-boolean-))
 - `http-client.max-redirects` - set the maximum amount of HTTP redirections to follow. A value of 0 (the default) prevents redirections from being followed.
 
+## Remote-file-syncer
```

Comment:
Minor. Remote File Syncer can be related to particular entity like `maxmind` etc. Pls add possible variations.

Reply:
I guess a *. prefix before each property will do?